src/components/swap/PendingModalContent/index.tsx

```diff
@@ -282,31 +300,24 @@ export function PendingModalContent({
         )}
         {/* Shown only during the final step under "success" conditions, and scales in. */}
         {currentStep === ConfirmModalState.PENDING_CONFIRMATION && showSuccess && <AnimatedEntranceConfirmationIcon />}
+        {/* Shown only during the final step on mainnet, when the transaction is sent but pending confirmation. */}
+        {currentStep === ConfirmModalState.PENDING_CONFIRMATION && showSubmitted && <AnimatedEntranceSubmittedIcon />}
         {/* Scales in for the USDT revoke allowance step if the revoke is pending onchain confirmation. */}
         {/* Scales in for the setup approval step if the approval is pending onchain confirmation. */}
         {/* Scales in for the final step if the swap is pending user signature or onchain confirmation. */}
-        {((currentStep === ConfirmModalState.PENDING_CONFIRMATION && !showSuccess) ||
+        {((currentStep === ConfirmModalState.PENDING_CONFIRMATION && !showSuccess && !showSubmitted) ||
           tokenApprovalPending ||
           wrapPending ||
           revocationPending) && <LoadingIndicatorOverlay />}
       </LogoContainer>
       <HeaderContainer
         gap="md"
-        $disabled={revocationPending || tokenApprovalPending || wrapPending || (swapPending && !showSuccess)}
+        $disabled={
+          revocationPending || tokenApprovalPending || wrapPending || (swapPending && !showSuccess && !showSubmitted)
```

Comment:
we use the `revocationPending || tokenApprovalPending || wrapPending` condition a couple times, can we pull this out to its own const